package.json

```diff
@@ -22,6 +23,7 @@
     "build": "npm run _rushRebuild",
     "buildci": "npm run _rushRebuildCi",
     "buildfast": "npm run _rushBuild",
+    "experiments": "npm run _rushBuildToExperiments",
```

Comment:
nit: I would suggest having this named a little different to be consistent with the already existing alias for starting `experiments` package in here:
https://github.com/OfficeDev/office-ui-fabric-react/blob/d3158b9b453f93ed5695390bd7d76eccd9b60a76/package.json#L21
Maybe something along the lines of `npm run build-exp`. `npm run experiments` sounds more like starting the project than building it, so `build-exp` might also help semantically.

Reply:
hmm. In the future, I expect the number of packages to increase. We just added datetime package; what would we do with that? `npm run datetime` ?

Perhaps a more scale-able solution would be to have `npm start` inside of experiments build --to experiments. I'm not sure. I'd want it to just ensure dependencies are built, which aren't built in webpack.

@kenotron thoughts?

Reply:
I think webpack.serve.config.js will need to have some better smarts about what it will alias.

Reply:
maybe we should have a generic `npm start` that would give you a menu to choose which project ... and then webpack would take care to compile the deps via aliasing.

Reply:
@dzearing @kenotron I could update this PR to just make the change in documentation so we at least have `buildfast` in there, and make a subsequent PR once we have come to an agreement on the correct way to do this.

What do you think of this approach?

Reply:
It's funny because rush abstracts npm and now we try to abstract rush inside npm. Okay, so the real requirement (user story) is this:

> devs are able to build experiments without them needing to have rush installed.

Let's slow down and think how we can achieve this in a scalable way. I think what @dzearing  suggested is for us to make a smarter "start" task so it can do the right thing. So I'm gonna chat with you offline to see if this is something we can tackle?